Dockerfile

```diff
@@ -11,7 +11,7 @@ COPY requirements.txt /pcw/
 # * Install pip requirements
 # * Empty system cache to conserve some space
 RUN zypper -n in python310-devel gcc libffi-devel && pip3.10 install -r /pcw/requirements.txt && rm -rf /var/cache
-
+RUN zypper in -y kubernetes1.23-client
```

Comment:
just include this in list above , this will decrease build time

Reply:
Build time and number of layers :smile: